internal/minipipeline/testdata/webconnectivity/generated/badSSLWithExpiredCertificate/measurement.json

```diff
@@ -28,17 +28,95 @@
   },
   "test_keys": {
     "agent": "redirect",
-    "client_resolver": "",
+    "client_resolver": "130.192.3.21",
     "retries": null,
     "socksproxy": null,
     "network_events": [
+      {
+        "failure": null,
+        "operation": "tls_handshake_start",
+        "t0": 0.011252,
+        "t": 0.011252,
+        "transaction_id": 3,
+        "tags": [
+          "depth=0",
+          "fetch_body=true"
+        ]
+      },
+      {
+        "failure": null,
+        "operation": "tls_handshake_done",
+        "t0": 0.02195,
+        "t": 0.02195,
+        "transaction_id": 3,
+        "tags": [
+          "depth=0",
+          "fetch_body=true"
+        ]
```

Comment:
There's another bug here: we're not adding the endpoint to `tls_handshake_xxx` and `http_transaction_xxx` events, which means that they do not sort correctly within the generated JSON.

Reply:
I will fix this bug as a separate pull request because it seems this one is already huge 😞

Reply:
Oh, and maybe instead I can just fix it here

Reply:
Okay, so, compromise: doing here for LTE, doing in the future for the DSLs.